nextcloudappstore/api/v1/release/info.xsd

```diff
@@ -357,6 +364,10 @@
                         maxOccurs="unbounded"/>
             <xs:element name="admin-section" type="php-class" minOccurs="0"
                         maxOccurs="unbounded"/>
+            <xs:element name="personal" type="php-class" minOccurs="0"
```

Comment:
Supported with 13+
https://github.com/nextcloud/server/pull/4890

Reply:
also updating the docs would be nice https://github.com/nextcloud/appstore/blob/master/docs/developer.rst#app-metadata

E.g. what interface to implement or which base class to extend and what it does

Reply:
done